app/models/tslr_claim.rb

```diff
@@ -37,18 +37,36 @@ class TslrClaim < ApplicationRecord
   validates :claim_school,              on: :"claim-school", presence: {message: "Select a school from the list"}
   validates :qts_award_year,            on: :"qts-year", inclusion: {in: VALID_QTS_YEARS, message: "Select the academic year you were awarded qualified teacher status"}
   validates :employment_status,         on: :"still-teaching", presence: {message: "Choose the option that describes your current employment status"}
-  validates :full_name,                 on: :"full-name", presence: {message: "Enter your full name"}
-  validates :address_line_1,            on: :address, presence: {message: "Enter your building and street address"}
-  validates :address_line_3,            on: :address, presence: {message: "Enter your town or city"}
-  validates :postcode,                  on: :address, presence: {message: "Enter your postcode"}
+
+  validates :full_name,                 on: :"full-name",
+                                        presence: {message: "Enter your full name"},
+                                        length: {maximum: 200, message: "Full name must be 200 characters or less"}
+
+  validates :address_line_1,            on: :address, presence: {message: "Enter your building and street address"},
+                                        length: {maximum: 100, message: "Address lines must be 100 characters or less"}
+
+  validates :address_line_2,            on: :address,
+                                        length: {maximum: 100, message: "Address lines must be 100 characters or less"}
+
+  validates :address_line_3,            on: :address, presence: {message: "Enter your town or city"},
+                                        length: {maximum: 100, message: "Address lines must be 100 characters or less"}
+
+  validates :postcode,                  on: :address, presence: {message: "Enter your postcode"},
+                                        length: {maximum: 11, message: "Postcode must be 11 characters or less"}
```

Comment:
Should we always validate these lengths, not just in the specific validation contexts? In the same way we did with the email address?

At the moment we could still get the DB error if we do something like:
```
TslrClaim.create!(postcode: '123456789123')
```